### ISSUE TITLE
Alternative auth method

### ISSUE THREAD

**Issue by cal0pteryx** [foobarjo/SqueezeSonic#4]
@foobarjo would you accept a pull request which adds an "Use legacy auth" switch to the plugin's settings? I'm trying to use SqueezeSonic in conjunction with Nextcloud's Music plugin, which doesn't support token-based auth, but only user/pw auth. Following my local tests, hardcoding user/pw based auth works fine so far. I'm willing to open a PR, but wanted to ask first, if this repo is still maintained :)

**Comment by flossgeek** [comment]
@wurstsalat3000 yes please! I'm also trying to connect this plugin to nextcloud, but can't get it working. I tried it with [this clunky workaround too](https://github.com/owncloud/music/wiki/Subsonic,-Workaround-for-using-DSub-5.4.4), but that didn't work out either. I would be very happy to see a solution!

**Comment by foobarjo** [comment]
Hi there,

The plug-in is not maintained anymore but feel free to send a PR I'll accept it.

Jo

**Comment by cal0pteryx** [comment]
Thank you!
